x-pack/test/api_integration/apis/security/index_fields.ts

```diff
@@ -33,7 +33,7 @@ export default function ({ getService }: FtrProviderContext) {
               'type',
               'visualization.title',
               'dashboard.title',
-              'search.columns',
+              'search.title',
```

Comment:
Do we know if there was a reason `search.columns` specifically was used here, or was it just the field that happened to be used? I'd like to drop it since it wasn't indexed anyway, and I can't imagine how it would be useful to restrict access to `columns` (does it even make sense to restrict access to saved object fields?).

Reply:
This change is from 2019, so it's unlikely we can figure out the exact reason (and it probably doesn't matter much now), but maybe @legrego still remembers?

Reply:
![image](https://media.giphy.com/media/l2JJO0D0JpgoU5OTe/giphy.gif)

I think this field just happened to be used. I don't see a problem replacing this with `search.title`

Reply:
Thanks for confirming @legrego!

Reply:
Sounds good, thanks for the input!